contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -332,9 +324,14 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         bytes32 originBlockchainID,
         TeleporterMessage calldata message
     ) external receiverNonReentrant {
+        bytes32 blockchainID_ = _initializeBlockchainID();
```

Comment:
don'pt think we need to intiailzie here, since we'll have to pass the check that a message was found, and can only pass if we've recieved a Teleporter message before, meaning blockchain id was initialized.

Reply:
I was thinking it would be a more descriptive error message in the case it hadn't been initialized yet, but you're right, "message not found" is just as descriptive because receiving the message will initialize the blockchain ID if necessary.

Reply:
I looked at this originally incorrect actually. I thought we could check whether the message was found first without the `blockchainID`, but blockchainID needs to be intiialized to calculate `messageID`. It still works since if `blockchainID` is not initialized, the messageID will be computed incorrectly. Can you add the same comment you added to `sendSpecifiedReceipts` to here?